web/storagenode/src/app/components/Bar.vue

```diff
@@ -8,36 +8,36 @@
 </template>
 
 <script lang="ts">
-    import { Component, Prop, Vue } from 'vue-property-decorator';
-
-    /**
-     * BarFillStyle class holds info for BarFillStyle entity.
-     */
-    class BarFillStyle {
-        'background-color': string;
-        width: string;
-
-        public constructor(backgroundColor: string, width: string) {
-            this['background-color'] = backgroundColor;
-            this.width = width;
-        }
+import { Component, Prop, Vue } from 'vue-property-decorator';
```

Comment:
Why did this indentation got removed?

Reply:
https://vuejs.org/v2/style-guide/?#Non-flux-state-management-use-with-caution

due to vue style guide indents inside script tag are not used. According to that new linter rule was added for such purposes earlier. 

Actually indents are not used inside style tag too, but fixing that would make PR too big. Should we decide to follow this or leave it be?

Reply:
I think making a different PR to fix the styling issues makes sense 👍